public/angular/app.js

```diff
@@ -0,0 +1,71 @@
+angular.module('fc', [
+   'ui.router', 
+   'fc.nav',
+   'fc.home',
+   'fc.comingSoon',
+   'fc.account.register'
+]).config(function($stateProvider, $urlRouterProvider) {
```

Comment:
In other instances you dropped the chained methods down a line, here you did not.